docs/src/dev-contributors.md

```
@@ -0,0 +1,145 @@
+
+<div class="contributors-grid" id="contributors-grid">
+  <!-- Contributors will be dynamically loaded here -->
+  <p class="loading-message">Loading contributors...</p>
+</div>
+
+<script>
+  document.addEventListener('DOMContentLoaded', async () => {
+    try {
+      // Fetch contributors from GitHub API
+      const response = await fetch('https://api.github.com/repos/truefoundry/elasti/contributors');
+      
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 6

(MD009, no-trailing-spaces)